.github/workflows/run-tests.yml

```diff
@@ -37,10 +37,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8, 3.9, "3.10", 3.11]
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: [3.8, 3.9, "3.10", 3.11, 3.12]
```

Comment:
It's petty, but what about having all these python versions in quotes so they look consistent?